conf.py

```diff
@@ -226,6 +231,44 @@
     # match any anchor that starts with a '/' since this is an invalid HTML anchor
     '\/.*',
 ]
+rediraffe_redirects = {
+    "clang.rst": "advanced-tools/clang.rst",
+    "coverity.rst": "advanced-tools/coverity.rst",
+    "gdb.rst": "advanced-tools/gdb.rst",
+    "coredev.rst": "core-developers/becoming-coredev.rst",
```

Comment:
And rename the file:

```suggestion
    "coredev.rst": "core-developers/becoming-core-dev.rst",
```